geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -1076,6 +1080,20 @@ private void configureClientSSLSocket(Socket socket, int timeout) throws IOExcep
 
       SSLParameters modifiedParams =
           checkAndEnableHostnameValidation(sslSocket.getSSLParameters());
+
+      String sysId = "";
+      InternalDistributedSystem system = InternalDistributedSystem.getAnyInstance();
+      if (system != null) {
+        sysId = system.getConfig().getClientHelloExtension();
```

Comment:
1. If this property is ssl related, it should start with "ssl-"
1. I don't see the connection between the 'client-hello-extension' and 'sysId', probably a better name for this property?
1. adding a config property probably would need a discussion thread on the dev list to gather feedback first.